packages/paste-core/components/list/__tests__/index.spec.tsx

```diff
@@ -267,3 +238,143 @@ describe("ListItem", () => {
+      );
+      const renderedList = screen.getByRole("list");
+      expect(renderedList).toHaveStyleRule("margin-top", "0.75rem");
+      expect(renderedList).toHaveStyleRule("margin-bottom", "0.75rem");
+    });
+
+    it("should allow aria, data, and html attributes", () => {
+      render(
+        <OrderedDisplayList aria-expanded="true" data-test="test-hook" title="random title">
+          <OrderedDisplayListItem aria-expanded="true" data-test="test-hook">
+            <OrderedDisplayListHeading aria-expanded="true" data-test="test-hook" as="h2">
+              Heading20
+            </OrderedDisplayListHeading>
+            Text beneath heading
+          </OrderedDisplayListItem>
+        </OrderedDisplayList>,
+      );
+      const renderedList = screen.getByRole("list");
+      expect(renderedList).toHaveAttribute("aria-expanded", "true");
+      expect(renderedList).toHaveAttribute("data-test", "test-hook");
+      expect(renderedList).toHaveAttribute("title", "random title");
+      const renderedListItem = screen.getByRole("listitem");
+      expect(renderedListItem).toHaveAttribute("aria-expanded", "true");
+      expect(renderedListItem).toHaveAttribute("data-test", "test-hook");
+      const renderedHeading = screen.getByRole("heading");
+      expect(renderedHeading).toHaveAttribute("aria-expanded", "true");
+      expect(renderedHeading).toHaveAttribute("data-test", "test-hook");
+    });
+  });
+
+  describe("Customization", () => {
+    it("should set a element data attribute for OrderedDisplayList", () => {
+      render(
+        <OrderedDisplayList>
+          <OrderedDisplayListItem>
+            <OrderedDisplayListHeading as="h2">Heading</OrderedDisplayListHeading>
+            Text
+          </OrderedDisplayListItem>
+        </OrderedDisplayList>,
+      );
+      expect(screen.getByRole("list").getAttribute("data-paste-element")).toEqual("ORDERED_DISPLAY_LIST");
+      expect(screen.getByRole("listitem").getAttribute("data-paste-element")).toEqual("ORDERED_DISPLAY_LIST_ITEM");
+      expect(screen.getByRole("heading").getAttribute("data-paste-element")).toEqual("ORDERED_DISPLAY_LIST_HEADING");
+    });
+    it("should set a custom element data attribute for OrderedDisplayList", () => {
+      render(
+        <OrderedDisplayList element="foo">
+          <OrderedDisplayListItem element="fooitem">
+            <OrderedDisplayListHeading element="foohead" as="h2">
+              Heading
+            </OrderedDisplayListHeading>
+            Text
+          </OrderedDisplayListItem>
+        </OrderedDisplayList>,
+      );
+      expect(screen.getByRole("list").getAttribute("data-paste-element")).toEqual("foo");
+      expect(screen.getByRole("listitem").getAttribute("data-paste-element")).toEqual("fooitem");
+      expect(screen.getByRole("heading").getAttribute("data-paste-element")).toEqual("foohead");
+    });
+
+    it("should add custom styles to OrderedDisplayList", (): void => {
```

Comment:
didn't you prove that we technically don't need these tests with customization VRT?

Reply:
I forgot I did that, been a while. Removing lol